dashboard/lib/tasks/seed.rake

```diff
@@ -107,7 +107,7 @@ namespace :seed do
   end
 
   # detect changes to .text_match.txt files
-  DSL_TYPES = %w(TextMatch ContractMatch)
+  DSL_TYPES = %w(TextMatch ContractMatch External)
```

Comment:
This looks good - in a future refactor we should be able to derive this list from the subclasses of BaseDSL (and roll the separate multi/match rake tasks into this one), so we don't need to keep this in sync when we add any other new dsl types.